### ISSUE TITLE
Re-export of default exports is not working properly (Star exports)

### ISSUE THREAD

**Issue by Schahen** [Kotlin/dukat#266]
```ts
import ping = require("a");
export = ping;
```
or even just
```ts
export = require("a");
```

Won't be translated at all. I can add a separate tag for empty translation but technically I'm not sinning agains the truth here when considering this unusable code.

**Comment by liorgonnen** [comment]
With the current version of Dukat, it's impossible to generate externals out of the box for three.js since the main (Three.d.ts) file is just a collection of exports.

The `Three.d.ts` file looks like this:
```
export * from './polyfills';
export * from './renderers/WebGLMultisampleRenderTarget';
export * from './renderers/WebGLCubeRenderTarget';
export * from './renderers/WebGLRenderTarget';
export * from './renderers/WebGLRenderer';
export * from './renderers/shaders/ShaderLib';
export * from './renderers/shaders/UniformsLib';
export * from './renderers/shaders/UniformsUtils';
// ...More exports...
```
Running Dukat on this file generates nothing.

**Comment by Schahen** [comment]
I'm splitting this task into to - as of star exports - this is done and will be released in 0.5.5, as of require - this actually needs more investigation.
